src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_debug.c

```diff
@@ -908,28 +999,30 @@ void KaleidoScope_UpdateDebugEditor(PlayState* play) {
                                                         (gSaveContext.save.skullTokenCount & 0xFFFF0000);
                 }
 
-            } else if (sCurSection == 70) {
+            } else if (sCurSection == INV_EDITOR_SECTION_NOTEBOOK) {
                 // Bombers Notebook
                 if (CHECK_BTN_ALL(input->press.button, BTN_CUP) || CHECK_BTN_ALL(input->press.button, BTN_CLEFT)) {
                     gSaveContext.save.inventory.questItems ^= gBitFlags[QUEST_BOMBERS_NOTEBOOK];
                 }
 
-            } else if (sCurSection == 69) {
+            } else if (sCurSection == INV_EDITOR_SECTION_LULLABY_INTRO) {
                 // Goron Lullaby Intro
                 if (CHECK_BTN_ALL(input->press.button, BTN_CUP) || CHECK_BTN_ALL(input->press.button, BTN_CLEFT)) {
                     gSaveContext.save.inventory.questItems ^= gBitFlags[QUEST_SONG_LULLABY_INTRO];
                 }
 
-            } else if (sCurSection <= 68) {
-                // Songs
-                slot = sCurSection - 51;
+            } else if (sCurSection < INV_EDITOR_SECTION_LULLABY_INTRO) {
+                // Boss Remains and Songs
+                //! Note: quest items must align with section, and all cases below INV_EDITOR_SECTION_LULLABY_INTRO must
+                //! have also been taken
```

Comment:
Not sure what this means.

Reply:
It's that the ordering of `sCurSection` starting from the boss remains must align with the ordering of `questItems`, or else the debugger breaks (i.e. don't reorder things in this section)